### ISSUE TITLE
[$30][MSFT-260] [Challenge Details] Compete, Tracks and Community menus can't access via arrow keys

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2909]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095347
2. Click near logo and use tab key to go to Tracks or Community
3. When user is in the Tracks menu
4. Press arrow (up/down) keys

**Expected Result**
Should be able to access the menus using arrow keys, if not user can't access those menus

**Actual Result**
Compete, Tracks and Community menus can't access via arrow keys

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6d6ec7be%2Dde44%2D4211%2Dbc02%2D19abb2187a9e&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=false&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F77

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095967 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Is this open for pickup?

**Comment by veshu** [comment]
@r0hit-gupta No, please wait for open for pickup label.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095967 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096733 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096733 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by afrisalyp** [comment]
This is related to two PR.
https://github.com/topcoder-platform/community-app/pull/2988
And
https://github.com/topcoder-platform/topcoder-react-utils/pull/42

**Comment by dmessing** [comment]
In Chrome, I can' use the arrow keys to go through the menu

**Comment by lakshmiathreya** [comment]
@nkumar-topcoder @sushilshinde - Didnt notice this coming for Dev QA Testing ... did Prod merge actually happen? Could you check ...

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

Note: Ticket was dependent on react Utils

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096733<br/><br/>```This is an automated message for crazyk via Topcoder X```
